rfcs/0001-notary-integration.md

```diff
@@ -0,0 +1,39 @@
+# Problem
+kpack does not handle image signing.
```

Comment:
I know it is a bit cumbersome but, can we have a bit more detail on why users might want image signing?